Missing form types (#1719)
PrestaShop/docs: development/components/form/types-reference/choice-category-tree.md

---
title: ChoiceCategoriesTreeType
---

# ChoiceCategoriesTreeType

> This form class is responsible to create a category selector using Nested sets

Reviewer: ```suggestion
This form class is responsible for creating a category selector using a tree view
```

would that work?
